docs/index.html

```diff
@@ -59,7 +59,7 @@
 
 
     <div id="article" class="container" style="margin-top:40px;max-width:800px; min-height:calc(100vh - 100px); padding-bottom:180px;">
-      <a href='https://prop.house/nouns/private-voting-research-sprint/3954'><div class="row postThumb">
+      <a href='/.html'><div class="row postThumb">
```

Comment:
I guess that this is due the usage of an older version of [blogo](https://github.com/arnaucube/blogo), which does not support posts from external places. If you can update your local blogo version ([v0_20221124](https://github.com/arnaucube/blogo/releases/tag/v0_20221124)).
If not, it's still fine because in theory the github bot that we have in this blog repo will re-generate the html files using latest version, so it should be fine.